jvector-base/src/main/java/io/github/jbellis/jvector/graph/disk/FusedADCNeighbors.java

```diff
@@ -25,4 +25,10 @@ public interface FusedADCNeighbors {
      * @return the quantized, transposed, and packed vectors of the given node's neighbors.
      */
     ByteSequence<?> getPackedNeighbors(int node);
+
+    /**
+     * Get the maximum degree of the graph for which this FusedADCNeighbors instance is used.
+     * @return the maximum degree of the graph.
+     */
+    int maxDegree();
```

Comment:
Are you planning to add more implementations of FADCN?  If not I'd prefer to just inline it into the implementation